cypress/integration/header.spec.ts

```diff
@@ -1,57 +1,38 @@
 /// <reference path="../support/index.d.ts" />
 
+const testNavButton = (buttonId: string, url: string) => {
+  cy.get(`[data-cy=nav-${buttonId}]`)
+    .click()
+  cy.url().should('eq', Cypress.config().baseUrl + `${url}`);
+}
+
+const createNavTest = (msg: string, buttonId: string, url: string) =>
+  () => it(msg, () => { testNavButton(buttonId, url) })
```

Comment:
Why is this function nested?

Reply:
Less code duplication. Since desktop and mobile both have the same navigation tests it's good practice to have them both call a function to execute the tests. However, this function would contain a lot of duplicated code since each nav-button test is identical. I'll make a change to make my intentions clearer.

Reply:
Resolved in a704b50